package.json

```diff
@@ -77,27 +77,16 @@
     "release": "npm run compile && sleep 4 && npm run bundle && sleep 4 && npm run publish",
     "postinstall": "electron-builder install-app-deps",
     "test": "npm run compile && npm run test:main && npm run test:components && npm run test:e2e",
-    "test:unit": "jest --detectOpenHandles --runInBand",
-    "test:main": "jest --env=node --detectOpenHandles --runInBand ./test/main",
-    "test:components": "jest --env=jsdom --detectOpenHandles --testTimeout=300 ./test/app",
+    "test:unit": "npm run test:unit:main && npm run test:unit:components",
```

Comment:
the point of this script was actually to be able to run ad hoc tests like `npm run test:unit -- <path to test(s)>` so let's keep that in there in some way

Reply:
try now

Reply:
Actually think the e2e tests are mislabelled and should be 'integration' but we've already done a fair amount of yak shaving in this PR so can consider that some other time